api/Document.json

```diff
@@ -1001,6 +1001,56 @@
           }
         }
       },
+      "blur_event": {
```

Comment:
A `Document` can't be the target of a "blur" event, so I don't think we should add this, or indeed any entries. We already have the needed "blur_event" and "focus_event" entries on `Element` and `Window`, which I think is the right place for them. So just removing the GlobalEventHandlers entries in this PR should do it.